Cabal/Distribution/Simple/Glob.hs

```diff
@@ -121,7 +127,7 @@ data Glob
 
 data GlobFinal
   = FinalMatch IsRecursive MultiDot String
-    -- ^ First argument: Is this a @**/*.ext@ pattern?
+    -- ^ First argument: Is this a @** / *.ext@ pattern?
```

Comment:
I had to change this to avoid CPP interpreting `/*` as a comment and failing with an 'unterminated comment' error.  I couldn't find any way to escape it or something, so I just went with this.

Reply:
Ideally, that file has no CPP at all; in fact, now that I see it, the conditional import of `Data.List.NonEmpty` looks supicous -- that's something that ought to be moved into the Compat.Prelude module which might make the use of CPP redundant